frontends/mrxl/fud/mrxl.py

```diff
@@ -32,7 +33,7 @@ def defaults():
         Specify defaults that should be added to fud's configuration file when
         this stage is registered.
         """
-        return {"exec": "mrxl"}
+        return {"exec": "mrxl", "data": None}
```

Comment:
This is only used when we [register a `fud` stage][fud-reg] and will not affect existing configurations. More importantly, you probably don't want this because there is no "default" data file that will work with all `mrxl` stages.